Vagrantfile

```diff
@@ -160,6 +160,7 @@ end
 VAGRANTFILE_API_VERSION = "2"
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     config.vm.box_check_update = false
+    config.vbguest.auto_update = false
```

Comment:
This breaks if the machine doesn't have vbguest.
You might want to use something such as:
`config.vbguest.auto_update = false if Vagrant.has_plugin("vbguest")`